docs/modules/deployment/pages/air-gap/air-gap-install.adoc

```diff
@@ -0,0 +1,151 @@
+
+= Air-Gap System Installation
+:description: Offline Installation directions for air-gapped environments
+
+Some scenarios for deploying OpenNMS {page-component-title} require that the installation run in an isolated or "air-gapped" network. This page provides a directions for an offline installation of {page-component-title} into such a network. As with any offline installation of software, the exact steps may vary depending upon the precise environment details. Before attempting, users should be familiar with the basic OpenNMS {page-component-title} installation steps from our xref:core/getting-started.adoc[Getting Started page].
+
+[[basic-process-outline]]
+== Basic Process Outline
+
+The basic process involves downloading the RPMs for OpenNMS {page-component-title} and any dependencies, transferring them to the air-gapped host, performing the installation, and then starting the install.
+
+## Prerequisites
+- A system with internet access to download packages
+- Target system running a compatible distribution
+- Sufficient disk space for package downloads
+- Administrative privileges on both systems
+
+
+## Part 1: Package Download (Internet-Connected System)
+
+### Step 1: Create a Temporary Package Directory
+
+Create a base directory to organize all downloaded packages.  This directory will become the base directory and dependencies will be included inside sub-folders to aid in knowing what order to install them.
+
+```bash
+mkdir ./install
+```
+
+### Step 2: Install OpenNMS Repository
+
+Add the {page-component-title} repository and import the GPG key:
+
+ifeval::["{page-component-title}" == "Horizon"]
+```bash
+dnf -y install https://yum.opennms.org/repofiles/opennms-repo-stable-rhel9.noarch.rpm
+rpm --import https://yum.opennms.org/OPENNMS-GPG-KEY
+```
+endif::[]
+
+ifeval::["{page-component-title}" == "Meridian"]
+```bash
+cat << EOF | sudo tee /etc/yum.repos.d/opennms-meridian.repo
+[meridian]
+name=Meridian for Red Hat Enterprise Linux and CentOS
+baseurl=https://REPO_USER:REPO_PASS@meridian.opennms.com/packages/2024/stable/rhel9
```

Comment:
s/`rhel9`/`rhel$releasever`/1 ?  Do we have a canonical Meridian repo config we send to customers?